.github/workflows/invitation.yml

```diff
@@ -12,6 +12,6 @@ jobs:
           organization: dev-protocol
           label: github-invitation
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-          comment: "![image](https://user-images.githubusercontent.com/73097560/138290996-69955e79-ccd2-4864-b416-8418f2c0a9ac.png)<br><b>Invitation sent to join the GitHub Organization. Welcome to Dev Protocol 🎉</b><br><br>Please check your mail to accept the invitation sent by the bot! 🤖<br><h1>About us</h1>Dev Protocol is a solution for decentralized creator economy, it is designed for the blockchain solution and enhance creators to create their economy 💻 Great to have you in our community ✨ Looking forward to your valuable contributions! 🤓 You can also check our community [socials](https://bio.link/devprotocol) ✨ 💛 You can also share our community with other folks who want to learn Blockchain, tech, and contribute together and with us 👉👈"
+          comment: "![image](https://raw.githubusercontent.com/dev-protocol/.github/main/assets/Dev--animated.gif)<br><b>Invitation sent to join the GitHub Organization. Welcome to Dev Protocol 🎉</b><br><br>Please check your mail to accept the invitation sent by the bot! 🤖<br><h1>About us</h1>Dev Protocol is a solution for decentralized creator economy, it is designed for the blockchain solution and enhance creators to create their economy 💻 Great to have you in our community ✨ Looking forward to your valuable contributions! 🤓 You can also check our community [socials](https://bio.link/devprotocol) ✨ 💛 You can also share our community with other folks who want to learn Blockchain, tech, and contribute together and with us 👉👈"
```

Comment:
What if we use this?

![image](https://user-images.githubusercontent.com/51391473/138764839-1011988e-66f2-4427-a8fc-e0380971df29.gif)

```suggestion
          comment: "![image](https://user-images.githubusercontent.com/51391473/138764839-1011988e-66f2-4427-a8fc-e0380971df29.gif)<br><b>Invitation sent to join the GitHub Organization. Welcome to Dev Protocol 🎉</b><br><br>Please check your mail to accept the invitation sent by the bot! 🤖<br><h1>About us</h1>Dev Protocol is a solution for decentralized creator economy, it is designed for the blockchain solution and enhance creators to create their economy 💻 Great to have you in our community ✨ Looking forward to your valuable contributions! 🤓 You can also check our community [socials](https://bio.link/devprotocol) ✨ 💛 You can also share our community with other folks who want to learn Blockchain, tech, and contribute together and with us 👉👈"
```

Reply:
Good!